.github/workflows/release.yml

```diff
@@ -243,3 +243,34 @@ jobs:
         $cred = New-Object System.Management.Automation.PSCredential $env:CLIENTID, $sec
         Set-StoreBrokerAuthentication -TenantId $env:TENANTID -Credential $cred
         Update-ApplicationSubmission -ReplacePackages -AppId 9NJNWW8PVKMN -SubmissionDataPath ".\Upload.json" -PackagePath ".\Upload.zip" -AutoCommit -Force
+
+  deploy-aur:
+    needs: [deploy-github-release]
+    runs-on: ubuntu-latest
+    if: startsWith(github.ref, 'refs/tags/')
```

Comment:
We should probably only run this for builds that we push into the release channel, right? So you would just add `environment: release-channel` as a line here, and then it is gated behind the release approval.

Reply:
Yes, it only runs for tags, but I think we want to gate it a bit more. I tag a lot, but many of these versions are crap and we don't want them to end up released. The environment stuff handles that, essentially.

Reply:
Sounds good. I have changed that now.